Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -1,13 +1,14 @@
 ## Table of Contents
 
-* [Install prerequisites](#install-prerequisites)
-* [Clone Oppia](#clone-oppia)
-* [Setup a virtual environment](#setup-a-virtual-environment)
-* [Running Oppia on a development server](#running-oppia-on-a-development-server)
-* [Tips and tricks](#tips-and-tricks)
-* [Notes on installation on Arch Linux systems](#notes-on-installation-on-arch-linux-systems)
-  * [Changes to installation prerequisites](#changes-to-installation-prerequisites)
-  * [Changes to the virtual environment setup](#changes-to-the-virtual-environment-setup)
+- [Table of Contents](#table-of-contents)
+- [Install prerequisites](#install-prerequisites)
+- [Clone Oppia](#clone-oppia)
+- [Setup a virtual environment](#setup-a-virtual-environment)
+- [Running Oppia on a development server](#running-oppia-on-a-development-server)
+- [Tips and tricks](#tips-and-tricks)
+- [Notes on installation on Arch Linux systems](#notes-on-installation-on-arch-linux-systems)
+  - [Changes to installation prerequisites](#changes-to-installation-prerequisites)
+  - [Changes to the virtual environment setup](#changes-to-the-virtual-environment-setup)
```

Comment:
Why this change? It looks like the only changes are (1) adding "Table of Contents" to the table of contents and (2) changing asterisks to hyphens. I don't think either change is necessary

Reply:
> Thanks @suyashmadhesia! It looks to me like the only substantive change here is to add `sudo apt-get update`. Am I reading it correctly? (it's fine if that's the only change that was needed; I just want to make sure I understand)

There is two changes:
1. sudo apt-get update
2. python-matplotlib -> python3-matplotlib

Reply:
> Why this change? It looks like the only changes are (1) adding "Table of Contents" to the table of contents and (2) changing asterisks to hyphens. I don't think either change is necessary

It was actually done by the markdown linter and I didn't notice that. I am updating my PR.